tests/add_fee_amount.go

```diff
@@ -0,0 +1,86 @@
+package tests
+
+import (
+	"context"
+	"math/big"
+
+	"github.com/ava-labs/subnet-evm/accounts/abi/bind"
+	teleportermessenger "github.com/ava-labs/teleporter/abi-bindings/go/Teleporter/TeleporterMessenger"
+	"github.com/ava-labs/teleporter/tests/network"
+	"github.com/ava-labs/teleporter/tests/utils"
+	localUtils "github.com/ava-labs/teleporter/tests/utils/local-network-utils"
+	"github.com/ethereum/go-ethereum/common"
+	. "github.com/onsi/gomega"
+)
+
+func AddFeeAmountGinkgo() {
+	AddFeeAmount(&network.LocalNetwork{})
+}
+
+func AddFeeAmount(network network.Network) {
+	subnets := network.GetSubnetsInfo()
+	subnetAInfo := subnets[0]
+	subnetBInfo := subnets[1]
+	teleporterContractAddress := network.GetTeleporterContractAddress()
+	fundedAddress, fundedKey := network.GetFundedAccountInfo()
+	ctx := context.Background()
+
+	subnetATeleporterMessenger, err := teleportermessenger.NewTeleporterMessenger(teleporterContractAddress, subnetAInfo.ChainRPCClient)
+	Expect(err).Should(BeNil())
+	subnetBTeleporterMessenger, err := teleportermessenger.NewTeleporterMessenger(teleporterContractAddress, subnetBInfo.ChainRPCClient)
+	Expect(err).Should(BeNil())
+
+	// Use mock token as the fee token
+	mockTokenAddress, mockToken := localUtils.DeployMockToken(context.Background(), fundedAddress, fundedKey, subnetAInfo)
```

Comment:
nit: one is deploy "mock" token, one is "exampel erc20", should they be same?

Reply:
Renamed `DeployMockToken` to `DeployExampleERC20`